api_tests/tests/ether_halight.ts

```diff
@@ -47,31 +47,17 @@ it(
         // Simulate that Bob is awaiting a swap from a different peer-id than Alice node's peer-id.
         bodies.bob.peer.peer_id =
             "QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N";
+        await alice.createSwap(bodies.alice);
         await bob.createSwap(bodies.bob);
-        await sleep(500);
 
-        await alice.createSwap(bodies.alice);
+        await assertSwapCreated(alice);
+        await assertSwapCreated(bob);
+
+        await sleep(1000);
 
-        const config = {
```

Comment:
This is a different test now.

We are no longer asserting the peer id check. We could do that as an internal test though and just get rid of this one?

Reply:
I don't agree, the test is equivalent.
But instead of checking for the availability of the actions (which seems quite an indirect way to check if the swap is finalized), we check the swap status.
If the swap status is still "created" then it means it has not been finalized.
When finalized, the swap status change to "In Progress".

I agree re internal check. I'll see if I get time to do it.

Reply:
Ah ok I missed that one, sorry about that :)